deployment/terraform/dataspace/main.tf

```diff
@@ -130,42 +132,82 @@ resource "azurerm_storage_account" "did" {
   static_website {}
 }
 
-resource "azurerm_key_vault_secret" "did_key" {
+resource "azurerm_key_vault_secret" "authority_did_key" {
   name = local.connector_name
   # Create did_key secret only if key_file value is provided. Default key_file value is null.
-  count        = var.key_file == null ? 0 : 1
-  value        = file(var.key_file)
-  key_vault_id = azurerm_key_vault.registry.id
+  count        = var.key_file_authority == null ? 0 : 1
+  value        = file(var.key_file_authority)
+  key_vault_id = azurerm_key_vault.registrationservice.id
```

Comment:
nit: Same as below comment, If registration service does not need a key vault, then maybe we do not deploy one for that and rename to `authority` as currently in this PR registration service key vault is used to store authority did private key only.

No action for now can be discussed in a follow-up PR.